.circleci/config.yml

```diff
@@ -18,39 +18,43 @@ jobs:
       - checkout: *post_checkout
       - restore_cache: *_root_package_lock_key
       - run: yarn install --frozen-lockfile
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./*
       - save_cache:
           <<: *_root_package_lock_key
           paths:
-            - "node_modules"
+            - ~/.cache/yarn
   
   lint:
     <<: *defaults
     steps:
-      - checkout: *post_checkout
-      - restore_cache: *_root_package_lock_key
+      - attach_workspace:
```

Comment:
Perhaps `attach_workspace` should be an anchor? The copy paste amount would be similar but if in the future we change the attach root (or any other `attach_workspace` options) we only have to change it in one place.

Reply:
Done.